osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerLoungeSubScreen.cs

```diff
@@ -112,37 +107,5 @@ protected override void OpenNewRoom(Room room)
 
             base.OpenNewRoom(room);
         }
-
-        private partial class MultiplayerListingPollingComponent : ListingPollingComponent
```

Comment:
This class was removed with no replacement. The general idea is that things generally handle if the API is online but the multiplayer client isn't connected.

It gets into this sort of state, where the create button is faded and clicking on the rooms has no effect:
<img width="754" alt="image" src="https://github.com/user-attachments/assets/91065b4e-566d-4d41-9417-42c20baf2879" />

We could potentially add an additional check for `MultiplayerClient.IsConnected` to the [main menu button](https://github.com/ppy/osu/blob/8c13a86ad1c51f0e3550d95ccd3a4443f2d15a53/osu.Game/Screens/Menu/ButtonSystem.cs#L182-L191) with perhaps an error notification, but I didn't see that as necessary.

Reply:
I think it's fine for now.